### DISCUSSION TITLE
API monitoring tools

### DISCUSSION THREAD

**Discussion by lgcyril** [DannyDainton/newman-reporter-htmlextra#439] [Q&A]
hello,

I am new to Postman/Newman and I'd like to know how to collect all the Newman reports htmlextra to import them in a tool (like Postman Monitor or else: which free tool?) to have a graph and historic of all reports in a nutshell?

Thank you in advance for your ideas.

Cyril 

PS: sorry if the question has already been discussed here, I didn't find anything yet about it :)

**Comment by DannyDainton** [comment]
All these files are external to Postman and there is no method in Postman to consume them and display them in the way you're after.

**Reply by DannyDainton** [reply]
You could use a different reporter like the json, csv or junit ones to get a raw output and then use another tool to consume that data? Never tried that so I wouldn't be able to give you any assistance here.
